integration-tests/hibernate-reactive-oracle/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+quarkus.datasource.db-kind=oracle
+quarkus.datasource.username=hibernate_orm_test
+quarkus.datasource.password=hibernate_orm_test
+
+# Hibernate config
+#quarkus.hibernate-orm.log.sql=true
+quarkus.hibernate-orm.database.generation=drop-and-create
+
+# Reactive config
+quarkus.datasource.reactive=true
+quarkus.datasource.reactive.url=${oracle.reactive.url}
```

Comment:
@IvanPetkov23 I think this should have been updated to `reactive-oracle.url` to match your other changes.

CI is failing because it wasn't.